Classes/Utility/AlertAction.swift

```diff
@@ -103,23 +103,23 @@ struct AlertAction {
     }
     
     static func discard(_ handler: AlertActionBlock? = nil) -> UIAlertAction {
-        return UIAlertAction(title: NSLocalizedString("Discard", comment: ""), style: .default, handler: handler)
+        return UIAlertAction(title: NSLocalizedString("Discard", comment: ""), style: .destructive, handler: handler)
     }
     
     static func toggleIssue(_ status: IssueStatus, handler: AlertActionBlock? = nil) -> UIAlertAction {
         let title = status == .open
             ? NSLocalizedString("Close", comment: "")
             : NSLocalizedString("Reopen", comment: "")
         
-        return UIAlertAction(title: title, style: .default, handler: handler)
+        return UIAlertAction(title: title, style: .destructive, handler: handler)
```

Comment:
Is reopening classified as a destructive measure? 🤔

Reply:
Ha, was thinking about this one. I *think* it would be better? As in: "this action has consequences". Was thinking to do it only for close tho.

Reply:
I get your point, but to me reopening isn't a destructive action. Red = "negative" actions. If anything this could be bold?

Reply:
From the docs:

> Apply a style that indicates the action might change or delete data.

Reply:
It's a tricky one - I lean towards destructive.

> As in: "this action has consequences"